### DISCUSSION TITLE
[Q&A] Is it possible to have multiple FL servers in real world application mode?

### DISCUSSION THREAD

**Discussion by jingyunc** [NVIDIA/NVFlare#2181] [Q&A] [answered]
### Python version (`python3 -V`)

3.8

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.4.0rc3

### Operating system

Ubuntu 20.04

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [ ] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

As my application has a peer-to-peer workflow, ideally I would like to make each client also a sever. I wonder if this is possible in NVFlare. Thanks.

**Comment by yhwen** [comment, marked as answer]
NVFlare does not support multiple FL servers. However, we do support peer-to-peer workflow. You can make use of the client side controller to control the workflow. Please checkout the swarm learning examples for reference.
